src/plugins/plugin.legend.js

```diff
@@ -421,9 +423,10 @@ module.exports = function(Chart) {
 						y: me.top + labelOpts.padding,
 						line: 0
 					};
+					cursor.y += (me.height - (itemHeight * me.legendItems.length)) / 2;
```

Comment:
why `+=` instead of `=`?

Reply:
agree. We could set this 2 lines above once.

Reply:
Agreed, I've done the suggested modification.